config/git_fetcher.go

```diff
@@ -32,7 +32,7 @@ func readGitConfig(configs ...*git.ConfigurationSource) (gf *GitFetcher, extensi
 			}
 
 			allowed := !gc.OnlySafeKeys
-			key, val := strings.ToLower(pieces[0]), pieces[1]
+			key, val := pieces[0], pieces[1]
```

Comment:
I think that a comment here is worthwhile. Something to the effect of: "don't re-case the incoming configuration key/value pairs from Git, since it is already done for us correctly".

Reply:
Sure, I can do that.